docs/releases/dev.rst

```diff
@@ -71,6 +71,10 @@ Command changes and additions
 - Added a :djadmin:`contributors` command to get the list of contributors
   (:issue:`3867`).
 
+- Added a :djadmin:`merge_user` command to get merge a user account to another
+
+- Added a :djadmin:`purge_user` command to purge a user's submissions
```

Comment:
Release notes are mainly to leave users know about changes, so providing a quick reasoning why these commands where introduced would be really good. For example `purge_user` is to kick evil users :)